lib/file.c

```diff
@@ -196,8 +198,33 @@ static CURLcode file_connect(struct Curl_easy *data, bool *done)
     return CURLE_URL_MALFORMAT;
   }
 
+  #if defined(AMIGA_FILESYSTEM)
```

Comment:
We usually go with just `ifdef` when checking a single define.

Reply:
Sorry - I'll fix that (this was originally part an `#elif`), but I moved it.